pkg/controllers/subnetset/subnetbinding_handler.go

```diff
@@ -0,0 +1,101 @@
+package subnetset
+
+import (
+	"context"
+	"time"
+
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/apis/vpc/v1alpha1"
+	"github.com/vmware-tanzu/nsx-operator/pkg/controllers/common"
+	servicecommon "github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
+)
+
+var (
+	removeFinalizerDelay = 1 * time.Second
+)
+
+func requeueSubnetSetByBindingMapUpdate(ctx context.Context, c client.Client, objOld, objNew client.Object, q workqueue.TypedRateLimitingInterface[reconcile.Request]) {
+	newBM, _ := objNew.(*v1alpha1.SubnetConnectionBindingMap)
+	if newBM.Spec.TargetSubnetSetName == "" {
```

Comment:
Do you mean user update SubnetBinding's target from TargetSubnetSetName to TargetSubnetName or from TargetSubnetName to TargetSubnetSetName?
It may happen, will update this logic.